package.json

```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "gulp __build && node ./index.js",
-    "production": "NODE_ENV=production node ./index.js",
+    "prod": "NODE_ENV=production node ./index.js",
```

Comment:
Changed to `prod` because yarn was doing weird things when it was named `production`.